### ISSUE TITLE
experimental backend crashes if shadow-exclude focused = 0

### ISSUE THREAD

**Issue by AeliusSaionji** [yshui/picom#483]
### Platform
archlinux 

### GPU, drivers, and screen setup
ancient intel laptop

mesa 20.1.7-1

```
name of display: :0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: Intel Open Source Technology Center (0x8086)
    Device: Mesa DRI Mobile Intel® GM45 Express Chipset (CTG) (0x2a42)
    Version: 20.1.7
    Accelerated: yes
    Video memory: 1536MB
    Unified memory: yes
    Preferred profile: compat (0x2)
    Max core profile version: 0.0
    Max compat profile version: 2.1
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 2.0
OpenGL vendor string: Intel Open Source Technology Center
OpenGL renderer string: Mesa DRI Mobile Intel® GM45 Express Chipset (CTG)
OpenGL version string: 2.1 Mesa 20.1.7
OpenGL shading language version string: 1.20

OpenGL ES profile version string: OpenGL ES 2.0 Mesa 20.1.7
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 1.0.16
```

### Environment
dwm

### picom version
**Version:** vgit-e553e

### Extensions:

* Shape: Yes
* XRandR: Yes
* Present: Present

### Misc:

* Use Overlay: Yes* Config file used: .cache/picom.conf

### Drivers (inaccurate):

modesetting

### Configuration:
```
shadow = true
shadow-exclude = [
	"focused = 0"
];
```

### Steps of reproduction
Use shadows + experimental backend, but exclude shadows for active window.

### Current Behavior

crash!

**Comment by tryone144** [comment]
Looks like a duplicate of https://github.com/yshui/picom/issues/465 which has already been fixed in `next`.
This fix is included in the v8.1 release which will probably coming to arch in the next days if you want to wait for it.

**Comment by AeliusSaionji** [comment]
ah, thanks.
It's hard to search this project's issues because of the template requesting an inline config- any item you search for is present in near every issue 😧
